eclass-writing/text.xml

```diff
@@ -266,8 +266,9 @@ summarizes the available documentation tags:
   <ti>Yes</ti>
   <ti>Single line freetext</ti>
   <ti>
-    Contains a short description for the eclass. Must be on the same
-    line with the tag.
+    Contains a short description for the eclass. Must be on the same line
+    with the tag. All words, including the first, must be in lowercase, except
```

Comment:
```suggestion
    as the tag. All words, including the first, must be in lowercase, except
```

Reply:
Well, that was part of the existing text. :smile: Updated.